apps/api/src/applicant/dto/ienapplicant-create.dto.ts

```diff
@@ -8,73 +8,112 @@ import {
   IsBoolean,
   IsArray,
 } from 'class-validator';
-import { IENApplicantCreateDTO } from '@ien/common';
+import { IENApplicantCreateUpdateDTO } from '@ien/common';
 
-export class IENApplicantCreateAPIDTO extends IENApplicantCreateDTO {
-  @ApiProperty({ description: 'Applicant Name', default: 'Mark Bowlill' })
+export class IENApplicantCreateUpdateAPIDTO extends IENApplicantCreateUpdateDTO {
+  @ApiProperty({ description: 'Applicant First Name', default: 'Mark' })
   @IsString()
-  @Length(1, 256, { message: 'Please provide applicant name' })
-  name!: string;
+  @Length(1, 256, { message: 'Please provide applicant first name' })
+  first_name!: string;
+
+  @ApiProperty({ description: 'Applicant Name', default: 'Bowlill' })
+  @IsString()
+  @Length(1, 256, { message: 'Please provide applicant last name' })
+  last_name!: string;
 
   @ApiPropertyOptional({
     description: 'Applicant unique ID',
-    default: 'a989765-c8789299-b9696669-dfcba98',
+    default: 1,
   })
-  @Length(1, 256, { message: 'Please provide applicant unique ID' })
   @IsOptional()
-  applicant_id?: string;
+  applicant_id?: number;
 
   @ApiPropertyOptional({
     description: 'Applicant email address',
     default: 'mark.bowill@mailinator.com',
   })
   @IsOptional()
-  email?: string;
+  @IsString()
+  @Length(1, 256, { message: 'Please provide applicant email' })
+  email_address?: string;
 
-  @ApiPropertyOptional({ description: 'Applicant citizenship', default: 'Maxico' })
+  @ApiPropertyOptional({
+    description: 'Applicant phone number',
+    default: '77-555-1234',
+  })
   @IsOptional()
-  citizenship?: string;
+  @IsString()
+  @Length(1, 256, { message: 'Please provide applicant phone' })
```

Comment:
don't think it's too big of a deal, but I think the max digits a phone number can have is 14 or 15? maybe cutting it down to something closer would help a user notice any errors more easily